services/galley/src/Galley/API/LegalHold.hs

```diff
@@ -42,6 +42,9 @@ createSettings (zusr ::: tid ::: req ::: _) = do
     assertLegalHoldEnabled tid
 
     membs <- Data.teamMembers tid
+    Log.debug $ Log.field "targets" (toByteString (show membs))
+              . Log.msg (Log.val "LegalHold.createSettings")
```

Comment:
I think using the `show` instance for `TeamMember` will create a log of noise:
```
data TeamMember = TeamMember
    { _userId          :: UserId
    , _permissions     :: Permissions
    , _invitation      :: Maybe (UserId, UTCTimeMillis)
    , _legalHoldStatus :: UserLegalHoldStatus
    } deriving (Eq, Ord, Show, Generic)
```
we should log the affected user id(s), comma separated.

Worth using a separate function altogether and move the logging details there? (i.e., the log level/field, etc.)

Reply:
oh yeh that was an oversight.  I should be logging the userIds here not the TeamMember